src/main/java/frc/robot/RobotContainer.java

```diff
@@ -113,6 +125,20 @@ private void configureButtonBindings() {
       .and(m_driverController.leftTrigger(OIConstants.kDriverControllerTriggerThreshold))
       .and(m_driverController.rightTrigger(OIConstants.kDriverControllerTriggerThreshold))
       .onTrue(new InstantCommand(() -> m_robotDrive.zeroHeading(), m_robotDrive));
+
+    //Intake Turn On
+    m_operatorController.leftTrigger(OIConstants.kOperatorControllerTriggerThreshold).onTrue(
```

Comment:
Looks good. Only slightly change to be made here is that you can chain onTrue and onFalse (I believe). No reason to use it except make it a little bit more readable (so one trigger is one line, and all the actions that the trigger can result in are together in the code)

Reply:
```
exampleButton
    // Binds a FooCommand to be scheduled when the button is pressed
    .onTrue(new FooCommand())
    // Binds a BarCommand to be scheduled when that same button is released
    .onFalse(new BarCommand());
```